src/eggd_generate_bed.sh

```
@@ -54,12 +54,12 @@ main() {
     # generate bed file
     if [[ -n ${optional_args+x} ]]; then
         if [[ -n "$additional_regions" ]]; then
-            python3 ~/generate_bed.py -p "$panel" -e ~/"$exons_name" -g ~/"$gene_panels_name" -t ~/"$genes2transcripts_name" -a ~/"$additional_regions_name" $optional_args
+            python3 ~/generate_bed.py -p "$panel" -e ~/"$exons_name" -g ~/"$gene_panels_name" -t ~/"$genes2transcripts_name" -a ~/"$additional_regions_name" -h "$header_info" $optional_args
```

Shellcheck (0.11.0)

[warning] exons_name is referenced but not assigned.

(SC2154)